public/wp-content/themes/coloredcow/package.json

```diff
@@ -1,6 +1,12 @@
 {
   "name": "ColoredCow",
   "version": "1.0.0",
```

Comment:
can we also change its version as we are upgrading it? Discuss it with others once.

Reply:
All agreed on upgrading the version to `1.0.1`.